python/nemo_relay/plugin.py

```
@@ -10,15 +10,9 @@
 from __future__ import annotations
 
-import asyncio
-import json
 import os
-import tomllib
-from collections.abc import Sequence
-from contextlib import asynccontextmanager
 from dataclasses import dataclass, field, fields, is_dataclass
-from pathlib import Path
-from typing import TYPE_CHECKING, AsyncIterator, Callable, Literal, Protocol, Self, TypedDict, cast
+from typing import TYPE_CHECKING, Callable, Literal, Protocol, Self, TypedDict, cast
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)